bot/kodiak/queries.py

```diff
@@ -114,6 +114,7 @@ class GraphQLResponse(TypedDict):
       body
       bodyText
       bodyHTML
+      url
```

Comment:
I'm surprised you didn't need to update `behind.json`

Reply:
don't worry, the tests diff fail :D